src/CodeGenerator/src/Generator/OperationGenerator.php

```diff
@@ -123,22 +123,24 @@ public function generate(Operation $operation): void
 
     private function setMethodBody(Method $method, Operation $operation, ClassName $inputClass, ?ClassName $resultClass): void
     {
-        $params = ['$response', '$this->httpClient'];
         if ((null !== $pagination = $operation->getPagination()) && !empty($pagination->getOutputToken())) {
-            $params = \array_merge($params, ['$this', '$input']);
-        }
+            $body = '
+                $input = INPUT_CLASS::create($input);
+                $response = $this->getResponse($input);
```

Comment:
```suggestion
                $response = $this->getResponse($input->request());
```

Reply:
Thta's my bad.. I didn't applyed your suggestion, but rewrite it by my won, and forgot this statement.

Fixed, and thanks for reporting